code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -155,3 +155,17 @@
 	icon_state = "griffinboots"
 	item_state = "griffinboots"
 	can_hold_items = 1
+
+/obj/item/clothing/shoes/gottagofast
+	name = "hedgehog's shoes"
+	desc = "The footwear of a mythical racer. Legends say that they invoke extreme speed in the wearer."
+	icon_state = "red"
+	slowdown = -INFINITY //yeah you gotta go fast

```

Comment:
The speed is actually capped on mobs. The crazy variable is like it is for silliness and to counteract all slowdowns.